test/functional/gh131.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const assert = require('assert');
+const crypto = require('crypto');
+
+const STORAGE = 'test-storage3';
+const TARBALL = 'blahblah';
+const PKG_NOCACHE = 'pkg_nocache';
+const PKG_NOCACHE2 = 'pkg_nocache2';
+
+function iscached(pkgname, tarballname) {
```

Comment:
camelCase here please, `isCache`.  The same in other functions. I'll enable camelCase eslint rule soon.